### PR TITLE
Desktop Version Project Files

### PR DESCRIPTION
**Pull request by iankiema**
Added the following files that contain my desktop version project;
-Index.html
-Style.css
-Images folder
-Readme file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -104,7 +105,6 @@ <h2>Profesional Art Printing Data</h2>
                       <li>Ruby</li>
                   </ul>
               </div>
-                <button type="button">See project</button>
             </div>
             <div class="showcase">
                 <div class="showcase-image">
```
Comment (topeogunleye) [inline index.html:124]: - [ ] As seen from the Figma Design. I kindly suggest that you ensure that:

- When the project is hovered on with the mouse, the description and dark opacity should disappear and a See Project button appears to the bottom of the image and take the whole image width 👍🏿

- When the project is hovered over, it should look exactly like the first project you have, while when your image is not hovered over, it should look like the rest of the projects.

image

File: index.html [hunk 2]
```diff
@@ -120,7 +120,7 @@ <h2>Profesional Art Printing Data</h2>
                       <li>Ruby</li>
                   </ul>
               </div>
-                <button type="button">See project</button>
```
Comment (topeogunleye) [inline index.html:123]: - [ ] I kindly suggest that your `See Project` button should link to the Live Demo of the project or to the GitHub repository.

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @iankiema, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
---

You have done a great job of creating a desktop version for template 2. Your project meets almost all the requirements. Your code is well-structured, clean and easy to read. Here are some specific points of feedback:

- Top Navigation: You have correctly hidden the hamburger button and aligned the item navigation and the student name navigation as required. You have also used semantic HTML tags and CSS flexbox to create a responsive layout. 👏
- Header/Headline: You have successfully aligned the text and the logo as required. You have also used appropriate font sizes, colors and weights to create contrast and hierarchy. Your social media icon links are well-placed and functional. 🙌
- Projects: You have created a clear and attractive section title with a line separator. You have also used CSS grid to create a responsive layout for the featured project and the regular projects. Your project images are high-quality and fit well with the design. Your project descriptions are concise and informative. You have also used button-like tags to display the technologies used for each project. Your See Project buttons are well-designed and link to the correct pages. 🚀
- About Myself: You have created a compelling section title with a larger font size. You have also written a clear and engaging introduction about your experience. Your Get My Resume button is well-designed and functional. You have used a grey line to separate the introduction from the columns of technologies. You have also used icons, headings and lists to display your language, framework and skill sets in an organized way. 💯
- Contact me: You have followed the important note and created one input for Full name instead of two. You have also aligned the bold text and the Get in touch button as required. Your form inputs are well-designed and functional. You have also used appropriate labels, placeholders and validations for your form. 📝

Overall, you have demonstrated excellent HTML and CSS skills and created a professional-looking desktop version for template 2

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- For future pull requests, please notice that your branch name `desktop-branch` does not follow the convention of being named after a feature or action. For example, you could name your branch `added-desktop-branch`. This makes it easier to track the changes and the purpose of each branch.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abdallahmalima — APPROVED** [review]
Hi @iankiema , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Correct use of media query✔️
- Use flexbox✔️
- Nice design✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._✔️

**Pull request merged**